agir/voting_proxies/tasks.py

```diff
@@ -7,6 +7,8 @@
 from agir.voting_proxies.models import VotingProxyRequest
 from agir.voting_proxies.links import VotingProxyLink
 
+SMS_SENDER = "Mélénchon 2022"
```

Comment:
```suggestion
SMS_SENDER = "Melenchon22"
```

on ne peut utiliser que l'éditeur validé par le presta, à la lettre près